Cargo.toml

```
@@ -105,6 +105,7 @@ futures-sink = { version = "=0.3.31", default-features = false }
 futures-util = { version = "=0.3.31", default-features = false }
 hashbrown = { version = "=0.15.0", default-features = false, features = ["nightly", "inline-more"] }
 http = { version = "=1.1.0", default-features = false }
+image = { version = "0.24.9", default-features = true }
```

`default-features = true` is the default and doesn't need writing down. Anyway, we want no default features:
```suggestion
image = { version = "0.24.9", default-features = false }
```

For `image`, the [default features](https://docs.rs/crate/image/latest/features) are `rayon` and `default-formats`. It's not documented, but [`rayon`](https://docs.rs/rayon/latest/rayon/) is most likely used to speed up processing by putting it into threads. I propose to test if the feature flag is worth enabling by comparing the time it takes to process images. `default-formats` is probably not needed. I think it's better to explicitly mention what formats are supported.

The features should be enabled in `libs/chonky/Cargo.toml`, so if other libraries in the repository depend on `image` it doesn't require the same feature set, e.g.:
```toml
image = { workspace = true, public = true, features = ["png"] }
```

---

I've worked with the image crate quite a few times in the past, and `rayon` only enables parallel iterators, about the `features`, it'd probably be worth it to have `0` features, `pdfium` directly pipes in RGBA data into a dynamic image, so doesn't need to decode anything.